src/pages/EntityDetails/Model/Model.js

```diff
@@ -69,8 +71,16 @@ const shouldShow = (segment, activeView) => {
   }
 };
 
+const generateCloudAndRegion = (cloudTag, region) => {
+  if (cloudTag && region) {
+    return `${extractCloudName(cloudTag)} / ${region}`;
+  }
+  return "";
+};
+
 const Model = () => {
-  const appState = useStore().getState();
+  const store = useStore();
+  const appState = store.getState();
```

Comment:
What's the point in this. Doesn't look like you use store anywhere else.

Reply:
Good catch, that was just a change from some investigation that ended up not being used. I've reverted that change.